hpy/tools/autogen/debug.py

```diff
@@ -88,8 +91,11 @@ def gen_debug_wrapper(self, func):
         #
         assert not func.is_varargs()
         node = get_debug_wrapper_node(func)
+        const_return = conf.RETURN_CONSTANT.get(func.node.name)
+        if const_return:
+            make_void(node)
         signature = toC(node)
-        rettype = toC(node.type.type)
+        rettype = 'void' if const_return else toC(node.type.type)
```

Comment:
Please use `get_context_return_type()`.

Reply:
`get_context_return_type` used to take the function object (and reads its `node.node.type`), but in this case we need to use the new `node` created at line 93. My take was that it's better to spare the callers of `get_context_return_type` from passing `func.node` instead of `func` and have with this one special case. I've generalized `get_context_return_type` so that it can be now used also here.